src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/XUnit/XUnitTestAssemblyFinishedCtorIntegration.cs

```diff
@@ -0,0 +1,49 @@
+// <copyright file="XUnitTestAssemblyFinishedCtorIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.XUnit
+{
+    /// <summary>
+    /// Xunit.Sdk.TestAssemblyRunner`1.RunAsync calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyNames = new[] { "xunit.execution.dotnet", "xunit.execution.desktop" },
+        TypeName = "Xunit.Sdk.TestAssemblyFinished",
```

Comment:
I struggled to find this type because it doesn't exist in the `main` branch... in 2.4.1 it's here: https://github.com/xunit/xunit/blob/2.4.1/src/messages/TestAssemblyFinished.cs, but looks like it's been renamed to `_TestAssemblyFinished` in latest (v3, not yet released)

Reply:
Yeah, for v3 the integration will not work, but not because that, right now the XUnitIntegration has `2.*.*` as a max version.